lib/rubycritic/rake_task.rb

```diff
@@ -0,0 +1,57 @@
+require "rake"
+require "rake/tasklib"
+
+module Rubycritic
+  class RakeTask < Rake::TaskLib
+    attr_accessor :name
+    attr_accessor :report
+    attr_accessor :format
+    attr_accessor :paths
+    attr_accessor :options
+
+    def initialize(*args, &task_block)
+      setup_ivars(args)
```

Comment:
"setup_ivars" is not the best name since it ties exactly what it does into the method name. Also I feel this indirection is pointless since it just "makes you look somewhere else" without adding real value.  How about moving this into the initializer?